website/assets/js/pages/try-fleet/register.page.js

```diff
@@ -0,0 +1,58 @@
+parasails.registerPage('register', {
+  //  ╦╔╗╔╦╔╦╗╦╔═╗╦    ╔═╗╔╦╗╔═╗╔╦╗╔═╗
+  //  ║║║║║ ║ ║╠═╣║    ╚═╗ ║ ╠═╣ ║ ║╣
+  //  ╩╝╚╝╩ ╩ ╩╩ ╩╩═╝  ╚═╝ ╩ ╩ ╩ ╩ ╚═╝
+  data: {
+    formData: { /* … */ },
+    // For tracking client-side validation errors in our form.
+    // > Has property set to `true` for each invalid property in `formData`.
+    formErrors: { /* … */ },
+
+    // Form rules
+    formRules: {
+      emailAddress: {required: true, isEmail: true},
+      password: {
+        required: true,
+        minLength: 12,
+        // Custom password validation to ensure it contains at least one letter, one number, and one special character. TODO: full list of special characters
+        custom: (password)=>{
+          return !! password.match(/[\!\@\#\$\%\^\&\*]/) && password.match(/\d/) && password.match(/\w/);
+        }
```

Comment:
Technically we don't need these stricter validations anymore, so we could revert it out save it for later, if needed. (This would avoid people who have already signed up from potentially having to change their password to make it valid.)